From 5b1869eaa22e365ab6595924fe96549b279b5ebc Mon Sep 17 00:00:00 2001
From: Olivier Brunel <jjk@jjacky.com>
Date: Fri, 20 Sep 2013 22:18:30 +0200
Subject: [PATCH 3/3] Fix possible lack of status messages on shutdown/reboot

Since 31a7eb86 the output on console can be disabled to avoid colliding with
gettys. However, it could also lead to a lack of messages during
shutdown/reboot.
---
 src/core/job.c | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/src/core/job.c b/src/core/job.c
index fc446fb..0dd161c 100644
--- a/src/core/job.c
+++ b/src/core/job.c
@@ -1097,6 +1097,9 @@ void job_shutdown_magic(Job *j) {
         if (!unit_has_name(j->unit, SPECIAL_SHUTDOWN_TARGET))
                 return;
 
+        /* In case messages on console has been disabled on boot */
+        j->unit->manager->no_console_output = false;
+
         if (detect_container(NULL) > 0)
                 return;
 
-- 
1.8.5.3

